corehq/apps/ota/utils.py

```diff
@@ -0,0 +1,58 @@
+from casexml.apps.case.xml import V2
+from casexml.apps.phone.restore import RestoreConfig, RestoreParams
+from corehq.apps.domain.models import Domain
+
+from .models import DemoUserRestore
+
+
+def turn_off_demo_mode(commcare_user):
+    """
+    Turns demo mode OFF for commcare_user and deletes existing demo restore
+    """
+    commcare_user.is_demo_user = False
+
+    # delete old restore
+    old_restore_id = commcare_user.demo_restore_id
+    if old_restore_id:
+        old_restore = DemoUserRestore.objects.get(uuid=old_restore_id)
+        old_restore.delete()
+
+    commcare_user.demo_restore_id = None
+    commcare_user.save()
+
+
+def turn_on_demo_mode(commcare_user, domain):
+    """
+    Turns demo mode ON for commcare_user, and resets restore to latest
+    """
+    # ToDo This should be in a task
```

Comment:
Demo users are not expected to have heavy restore payloads (project admins usually create these profiles to Demo), so may be it's okay if it's not a task? (I also pinged Sheel on spec about this)